tasks-by-users/Naufil-Shemle/01-e-commerce-website/index.html

```diff
@@ -0,0 +1,410 @@
+				<i class="bi bi-star-fill"></i>
+				<i class="bi bi-star-fill"></i>
+				<i class="bi bi-star-fill"></i>
+				<i class="bi bi-star-half"></i>
+				<i class="bi bi-star"></i>
+			</div>
+			<p>$50.00</p>
+		</div>
+
+		<div class="col-4">
+			<img src="images/product-7.jpg" alt="Product image">
+			<h4>Red Printed T-Shirt</h4>
+			<div class="rating">
+				<i class="bi bi-star-fill"></i>
+				<i class="bi bi-star-fill"></i>
+				<i class="bi bi-star-fill"></i>
+				<i class="bi bi-star-fill"></i>
+				<i class="bi bi-star-half"></i>
+			</div>
+			<p>$50.00</p>
+		</div>
+
+		<div class="col-4">
+			<img src="images/product-8.jpg" alt="Product image">
+			<h4>Red Printed T-Shirt</h4>
+			<div class="rating">
+				<i class="bi bi-star-fill"></i>
+				<i class="bi bi-star-fill"></i>
+				<i class="bi bi-star-fill"></i>
+				<i class="bi bi-star-fill"></i>
+				<i class="bi bi-star"></i>
+			</div>
+			<p>$50.00</p>
+		</div>
+	</div><div class="row">
+		<div class="col-4">
+			<img src="images/product-9.jpg" alt="Product image">
+			<h4>Red Printed T-Shirt</h4>
+			<div class="rating">
+				<i class="bi bi-star-fill"></i>
+				<i class="bi bi-star-fill"></i>
+				<i class="bi bi-star-fill"></i>
+				<i class="bi bi-star-fill"></i>
+				<i class="bi bi-star-fill"></i>
+				<i class="bi bi-star"></i>
+			
+			</div>
+			<p>$50.00</p>
+		</div>
+
+		<div class="col-4">
+			<img src="images/product-10.jpg" alt="Product image">
+			<h4>Red Printed T-Shirt</h4>
+			<div class="rating">
+				<i class="bi bi-star-fill"></i>
+				<i class="bi bi-star-fill"></i>
+				<i class="bi bi-star-fill"></i>
+				<i class="bi bi-star-half"></i>
+				<i class="bi bi-star"></i>
+			</div>
+			<p>$50.00</p>
+		</div>
+
+		<div class="col-4">
+			<img src="images/product-11.jpg" alt="Product image">
+			<h4>Red Printed T-Shirt</h4>
+			<div class="rating">
+				<i class="bi bi-star-fill"></i>
+				<i class="bi bi-star-fill"></i>
+				<i class="bi bi-star-fill"></i>
+				<i class="bi bi-star-fill"></i>
+				<i class="bi bi-star-half"></i>
+			</div>
+			<p>$50.00</p>
+		</div>
+
+		<div class="col-4">
+			<img src="images/product-12.jpg" alt="Product image">
+			<h4>Red Printed T-Shirt</h4>
+			<div class="rating">
+				<i class="bi bi-star-fill"></i>
+				<i class="bi bi-star-fill"></i>
+				<i class="bi bi-star-fill"></i>
+				<i class="bi bi-star-fill"></i>
+				<i class="bi bi-star"></i>
+			</div>
+			<p>$50.00</p>
+		</div>
+	</div>
+</div>
+
+
+<!-- -----------------Offer---------------- -->
+<div class="offer">
+	<div class="small-container">
+		<div class="row">
+			<div class="col-2">
+				<img src="images/exclusive.png" class="offer-img" alt="Exclusive Product Image">
+			</div>
+			<div class="col-2">
+				<p>Exclusively Available on RedStore</p>
+				<h1>Smart Band 4</h1>
+				<small>The Mi Band 4 featrue a 39.9% larger (than Mi Band 3) AMOLED color full-touch display with adjustable brightness, so everything is clear as can be.</small>
```

Comment:
What does `small` tag do ?

Reply:
It use to define smaller text, like copyright and side-comments etc.